etc/config/cuda.amazon.properties

```diff
@@ -12,108 +12,164 @@ group.nvcc.compilerType=nvcc
 group.nvcc.isSemVer=true
 group.nvcc.baseName=NVCC
 group.nvcc.includeFlag=-I
-group.nvcc.objdumper=/opt/compiler-explorer/cuda/11.7.0/bin/nvdisasm
+group.nvcc.nvdisasm=/opt/compiler-explorer/cuda/11.7.0/bin/nvdisasm
 group.nvcc.rpathFlag=-L # WAR, really need `-Xcompiler "-Wl,-rpath=<path>"`, but not required because supportsExecute=false
 group.nvcc.options=--keep-device-functions
-group.nvcc.demangler=/opt/compiler-explorer/cuda/11.3.1/bin/cu++filt
 compiler.nvcc91.semver=9.1.85
 compiler.nvcc91.exe=/opt/compiler-explorer/cuda/9.1.85/bin/nvcc
 compiler.nvcc91.options=--compiler-bindir /opt/compiler-explorer/gcc-6.4.0/bin
-compiler.nvcc91.objdumper=/opt/compiler-explorer/cuda/9.1.85/bin/nvdisasm
+compiler.nvcc91.nvdisasm=/opt/compiler-explorer/cuda/9.1.85/bin/nvdisasm
+compiler.nvcc91.demangler=/opt/compiler-explorer/gcc-6.4.0/bin/c++filt
+compiler.nvcc91.objdumper=/opt/compiler-explorer/gcc-6.4.0/bin/objdump
 compiler.nvcc92.semver=9.2.88
 compiler.nvcc92.exe=/opt/compiler-explorer/cuda/9.2.88/bin/nvcc
 compiler.nvcc92.options=--compiler-bindir /opt/compiler-explorer/gcc-6.4.0/bin
-compiler.nvcc92.objdumper=/opt/compiler-explorer/cuda/9.2.88/bin/nvdisasm
+compiler.nvcc92.nvdisasm=/opt/compiler-explorer/cuda/9.2.88/bin/nvdisasm
+compiler.nvcc92.demangler=/opt/compiler-explorer/gcc-6.4.0/bin/c++filt
```

Comment:
wow that's an old c++filt - is there a reason it needs to be quite so old? I assume so else you wouldn't have gone to quite the effort level needed to make the changes (sorry for it being so painful).

Reply:
I chose the `c++filt` from the same toolchain as we were already using for [`objdump`](https://github.com/trxcllnt/compiler-explorer/blob/10bd079d26aa4cf36f9e6a05072a7af8593e71a2/etc/config/cuda.amazon.properties#L29). I figure we had these set because the old CTK's all need to use gcc-6?

Reply:
Mayybe!? But either way, keeping the same as before is a sensible default :+1: